doc/tutorial/Docs.lhs

```diff
@@ -220,12 +220,10 @@ api :: Proxy DocsAPI
 api = Proxy
 
 server :: Server DocsAPI
-server = Server.server3 :<|> serveDocs
-
-  where serveDocs _ respond =
-          respond $ responseLBS ok200 [plain] docsBS
-
-        plain = ("Content-Type", "text/plain")
+server = Server.server3 :<|> Tagged serveDocs where
```

Comment:
It's a bit ugly to have `Tagged` appear here, but I don't have a better idea. If you've got one, I'm all ears.